lib/college/schemas/student.ex

```diff
@@ -1,13 +1,25 @@
 defmodule College.Schemas.Student do
   use Ecto.Schema
   import Ecto.Changeset
+  alias College.App
+  alias College.Repo
+  alias __MODULE__
+
+  defimpl College.Person do
+    def get(student) do
+      student |> Student.fill_virtual_fields()
+    end
+  end
```

Comment:
Usually we put the protocol implementations after schema definition.
Also, we don't use pipe for single line, please add it as Student.fill_virtual_fields(student)